pages/community/ambassador-program/index.js

```diff
@@ -0,0 +1,305 @@
+import React from 'react';
+import Button from '../../../components/buttons/Button';
+import GenericLayout from '../../../components/layout/GenericLayout';
+import Heading from '../../../components/typography/Heading';
+import IconRocket from '../../../components/icons/Rocket';
+import NewsletterSubscribe from '../../../components/NewsletterSubscribe';
+
+const data = [
```

Comment:
Please make these array variables in a separate file and import it from there so that if we update/add more elements to it, it won't clutter the main Dashboard implementation from the values we are using as arrays.

Reply:
Please ignore this kind of review for now. The reason for this is just for quick prototyping :)

Reply:
But if we ignore it right now and move these changes for final review, it will be very much difficult for me to review around 3k-4k lines of code in terms of changes, so it's better if we do the reviews in small PRs also, so that the burden for final PR won't be much.